webextension/manifest.json

```diff
@@ -10,10 +10,27 @@
   "author": "Mozilla (https://mozilla.org/)",
   "background": {
     "scripts": [
+      "study.js",
       "background.js"
     ]
   },
   "experiment_apis": {
+    "prefs": {
+      "schema": "./privileged/prefs/schema.json",
```

Comment:
You don't need this entry.  We don't have the Prefs api complete.

Reply:
Updated to remove prefs api https://github.com/mozilla/shield-cloudstorage/pull/38